interpreter/for_test.go

```diff
@@ -335,6 +335,7 @@ func TestInterpretForStatementCapturing(t *testing.T) {
 
 	t.Parallel()
 
+	// TODO: Use compiler (parseCheckAndPrepare)
```

Comment:
This test captures loop-variable as a closure. It seems, here it is expected to be behaved differently to other closures / everywhere else. Not sure how to implement this, so left it for now. Would be good to discuss/handle this separately.

Reply:
Yeah, this still needs to be implemented. It is already tracked as "Captured loop variables" in #3804

Reply:
oh I see, great!